<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QForm_prom_display</class>
 <widget class="QWidget" name="QForm_prom_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1028</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1028</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>استعلام عن العروض</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(198, 207, 199);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="Qcombo_company">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Qcombo_branch">
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Qcombo_cust_group">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Qcombo_sponsor">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>97</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>الشركة</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>الفرع</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="QcheckBox_customer_group">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>مجموعه العملاء</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="QcheckBox_sponsor_prom">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>ممول العرض</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="QcheckBox_magazine">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>140</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>عرض مجلة</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Qcombo_magazine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="title">
      <string>الحالة</string>
     </property>
     <widget class="QCheckBox" name="QcheckBox_magazine_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>15</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>فعال</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="QcheckBox_magazine_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>15</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>موقوف</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="QcheckBox_magazine_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>15</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>منتهي</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="QcheckBox_magazine_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>الكل</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="Qcombo_sponsor_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QCheckBox" name="QcheckBox_department">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>110</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>القسم</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="Qline_promotion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: gray;
    selection-background-color: darkgray;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="QgroupBox_prom">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>405</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>91</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="title">
       <string>فترة العرض</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>التاريخ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="Qdate_from">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2020</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>الوقت</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="Qtime_from">
          <property name="displayFormat">
           <string>h:mm AP</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>26</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>من</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>26</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>الى</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>التاريخ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="Qdate_to">
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2020</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>الوقت</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="Qtime_to">
          <property name="displayFormat">
           <string>h:mm AP</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="QcheckBox_magazine_9">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>التاريخ</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="Qcombo_promotion">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>نوع العرض</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioBtnPromNum">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>رقم العرض</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Qbtn_search">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>بحث</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_print">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: green;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>طباعة</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_exit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>خروج</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_8</zorder>
   <zorder>groupBox_3</zorder>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>1001</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QTableWidget" name="Qtable_promotion">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="columnCount">
       <number>33</number>
      </property>
      <column>
       <property name="text">
        <string>PROM_ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_LINE_NO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>POS_ITEM_NO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>POS_GTIN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>BMC_ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SPONSER_ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_PRICE_BEFORE_DISC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_ITEM_SCALE_FLAG</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_GROUP_SCALE_FLAG</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_DISCOUNT_FLAG</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_ITEM_DISCOUNT</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_GROUP_NO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_GROUP_DISC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_GROUP_ITEM_ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_GROUP_ITEM_DISC_SHARE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_ITEM_QTY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_ITEM_DISC_VAL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_ITEM_PRICE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_MAX_DISC_QTY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_MAX_DISC_VAL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_START_DATE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_END_DATE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_CREATED_ON</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_CHANGED_BY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_ACTIVATED_BY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_STOPPED_BY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_STATUS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_RESERVED1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_RESERVED2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_RESERVED3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_RESERVED4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_RESERVED5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROM_RESERVED6</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
